README.md

```diff
@@ -130,6 +130,28 @@ To use a different `Typescript TSDK` version than the one that comes with vscode
 }
 ```
 
+## Using Exponentjs
+
+We support using exponentjs to run, debug and publish your applications.
+
+When you use exponent for the first time in vscode (either to run, debug or publish) you will notice that we created a couple of files for you.
+
+Under your `.vscode/` directory there will be a `vscodeExponent.json` file. This file has the settings used for us to setup exponent correctly.
+
+```
+{
+    "entryPointFilename": ""          // File under the project root that is used as an entrypoint. We look for index.ios.js or index.android.js
+    "entryPointComponent": ""         // Name of the main component used in your entrypoint. By default it's the same name of your app
```

Comment:
"same name of your app" should be "same name *as* your app" or "name of your app"